_docs/development.md

```diff
@@ -25,23 +25,26 @@ You'll need the following to get started:
 ## Create temba user for PostgreSQL
 
 {% highlight bash %}
-$ createuser temba --pwprompt -d
+$ sudo apt-get install postgresql-client postgresql-server-dev-9.3 postgis*
+$ sudo -u postgres createuser temba --pwprompt -d
 Enter password for new role:
 Enter it again:
+$ sudo adduser temba
 {% endhighlight %}
 
 ## Create temba database, add PostGIS
 
 Create the database as temba user:
 {% highlight bash %}
-$ psql --user=temba postgres
+$ sudo -u temba psql --user=temba postgres
 postgres=> create database temba;
```

Comment:
I guess there are two options one is to list dependencies and let people try and figure it out.  The second is to say here are the install instruction for such-and-such distribution (e.g. Ubuntu 14.04 LTS).  For people that don’t know their way around unix, this later is a good option - otherwise you have a barrier for people getting RapidPro up and running.

If you say that you are only going to provide instal for LTS releases, you won’t have that much of a maintenance burden.
 
Cheers,
-carl



> On Jan 29, 2015, at 2:06 PM, Nic Pottier <notifications@github.com> wrote:
> 
> In _docs/development.md <https://github.com/rapidpro/rapidpro/pull/124#discussion_r23793379>:
> 
> >  {% endhighlight %}
> >  
> >  ## Create temba database, add PostGIS
> >  
> >  Create the database as temba user:
> >  {% highlight bash %}
> > -$ psql --user=temba postgres
> > +$ sudo -u temba psql --user=temba postgres
> >  postgres=> create database temba;
> Maybe, but that would be an issue with how you have Postgres installed. The problem is that a lot of these little things are dependent on your OS, how you have the DB installed and configured etc, that's why I'm hesitant to put in some of these changes because they are specific to your environment. (ie, these work as written on our environments)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/rapidpro/rapidpro/pull/124/files#r23793379>.
>